core/src/main/java/io/confluent/rest/ApplicationServer.java

```diff
@@ -384,11 +431,38 @@ private void configureConnectors(SslContextFactory sslContextFactory) {
     for (URI listener : listeners) {
       log.info("Adding listener: " + listener.toString());
       NetworkTrafficServerConnector connector;
-      if (listener.getScheme().equals("http")) {
-        connector = new NetworkTrafficServerConnector(this, httpConnectionFactory);
+
+      // Default to supporting HTTP/2 for Java 11 and later
+      if (IS_JAVA11_COMPATIBLE) {
```

Comment:
I think we should have a way to opt out of this new default if we end up hitting some problems with it in some circumstances. I am not sure what would be the most idiomatic way to do this, but having in mind that we don't expect to be using it extensively, I think a custom `RestConfig` property or maybe even a system property can do the job.

Reply:
+1 to flag protect. I don't think we will have any problem with having it on my default , but it costs us nothing to flag protect.

Reply:
Flag added to config. Defaults to true.